workshop/exercise-4/README.md

```diff
@@ -44,30 +62,26 @@ Because Knative is built on top of Kubernetes, you can use kubectl along with a
 
     Note: To exit the watch, use `ctrl + c`.
 
-4. Let's try out our new application! First, let's get the domain name that Knative assigned to the Service we just deployed. Run the following command, and note the value for `domain`. IBM Cloud Kubernetes Service sets the default domain name for Knative to match the domain name of your IBM Cloud Kubernetes Service cluster. It will also set up the Istio ingress to route all incoming requests targeted to that domain to Knative.
+4. Let's try out our application again! Because the service name was the same as the application you deployed before, `fib-knative`, the domain name for our service should be the same. You can double check if you want.
 
     ```
-    kubectl get ksvc fib-knative
+    kn service get fib-knative
```

Comment:
same output

Reply:
creating an issue for general sample output.